fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -59,6 +65,7 @@
      */
     public FedoraEvent(final Event e) {
         checkArgument(e != null, "null cannot support a FedoraEvent!");
+        eventID = pidMinter.mintPid();
```

Comment:
Yes, or they might want a different ID scheme for some other reason. Injection (with an appropriate default) is a good move here, I think.